xilem/src/view/button.rs

```diff
@@ -60,8 +69,8 @@ where
         _ctx: &mut ViewCtx,
         mut element: Mut<Self::Element>,
     ) {
-        if prev.label != self.label {
-            widget::Button::set_text(&mut element, self.label.clone());
+        if prev.label.label != self.label.label {
+            widget::Button::set_text(&mut element, self.label.label.clone());
```

Comment:
I *think* this might be able to use `self.label.rebuild(prev.label, /* state */, ctx, widget::Button::label_mut(&mut element))`

Absolutely this needs to handle all of the properties properly.

Reply:
It would be great to write self.label.rebuild, but rebuild method is not public. 

Now I am looking for solution similar as for sized_box, but it is not easy for me to figure out do I really need to rewrite code with something like V: WidgetView and etc. or there is more simple way.